content/en/blog/trusted-boot.md

```diff
@@ -0,0 +1,155 @@
+
+
+## From Grub to systemd-boot
+
+![DALL·E 2024-01-18 18 22 47 - Create a vintage-style digital artwork that visualizes the concept of modernizing technology infrastructure  The central focus should be a retro-futur](https://github.com/kairos-io/kairos-docs/assets/2420543/df3bce6f-4e88-4edd-83b2-ba2d333d9e6f)
+
+Trusted Boot is deeply integrated in systemd and currently under active development. Systemd is renowned for being more than a mere init system. It includes a range of tools like systemd-measure and ukify, which, though compatible with various bootloaders, synergize exceptionally well with systemd-boot. This makes a compelling case for choosing systemd, and particularly systemd-boot.
+
+Why this makes sense: We're now in an era where outdated hardware is not our concern. Modern devices don't require old drivers, so we have the opportunity to streamline our systems. This translates to a principle of 'less is more' in our code and in our stack - less software in our stack means enhanced security. We aim for a leaner runtime to run our software, minimizing potential security vulnerabilities. In addition, it also makes sense to use tools that adhere closely to standard specifications.
+
+Considering the transition to modern hardware, especially with widespread EFI support, systemd-stub or systemd-boot is a strong contender to replace GRUB. GRUB has been a staple in the Linux community for years, evolving from versions 0.x to 2.x(and if you are old like me, you have witnessed almost all of the versions!). However, it's worth considering the extensive patches applied by distributions and its expansive codebase. While its extensive support and scripting capabilities were once advantageous, they now detract from its suitability as a lightweight option.
+
+The goal is clear: we aim to minimize complexity in building a secure, efficient stack, reusing as much components well trusted and established by the community. By reducing the number of components, we strive for a more secure and streamlined system.
+
+## Booting in Linux
+
+When a Linux system starts, it goes through a multi-stage process to load the operating system. This process can be broken down roughly into the following steps:
+
+1. Firmware Initialization: The system's firmware, which could be BIOS or UEFI, performs initial hardware checks and configurations. It then searches for a bootloader, which is typically located in the boot partition or the Master Boot Record (MBR) of the storage device.
+1. Bootloader Execution: The bootloader, such as GRUB, is a small program responsible for loading the main part of the operating system. Its primary task is to find the Linux kernel, usually located in the same partition, and load it into memory. The bootloader might present a menu or configuration options to choose different kernels or operating systems if multiple are installed.
```

Comment:
I would remove the "small program" to avoid a counter argument to what was said before